src/Providers/YFrameServiceProviders.php

```diff
@@ -64,4 +66,33 @@ private function resolveCommands()
             ]);
         }
     }
+
+    protected function registerDirectives()
```

Comment:
you must add a comment block here

Reply:
it's already added!
![image](https://user-images.githubusercontent.com/37983260/152598046-52f45086-9379-42d2-841f-bb723d77b790.png)